LondonBritishLibrary/orient/BLorient9798.xml

```diff
@@ -0,0 +1,281 @@
+                                the letter <foreign xml:lang="gez-tr">ṣa/ḍa</foreign> has 9 verses,
+                                the letter <foreign xml:lang="gez-tr">fa</foreign> has 9 verses.</note>
+                            <incipit xml:lang="gez"><lb n="1"/>እደ፡ ሰብእናሁ፡ በሰናስለ፡ ንዴት፡ ተሞቅሐ። 
+                                <lb n="2"/>ማኅቶተ፡ ጥበብ፡ በእደ፡ ልቡ፡ ፈትሐ።
+                                <lb n="3"/>ሰይፈ፡ ጸሎት፡ እምቀቀብ፡ ልቡ፡ መልሐ።
+                                <lb n="4"/>በእግረ፡ ልቡ፡ ፍኖተ፡ ጥበብ፡ ተመርሐ።
+                                <lb n="5"/>እምፍኖተ፡ ዕበድ፡ ውስተ፡ ትምህርት፡ ጼሐ።</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i1.3">
+                            <locus from="39va" to="39vb"/>
+                            <title ref="LIT3975FekkareF"/>
+                            <incipit xml:lang="gez">ሀ፡ ብ፡ ሀልዎቱ፡ ለአብ፡ እምቅድመ፡ ዓለም።
+                                ለ፡ ብ፡ ለብሰ፡ ሥጋ፡ ቃል፡ እምድንግል።
+                                ሐ፡ ብ፡ ሐመ፡ ወሞተ፡ በእንቲአነ።
+                                መ፡ ብ፡ መንክር፡ እግዚአብሔር።
+                            </incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="39vb" to="49ra"/>
+                            <title ref="LIT7037Qene"/>
+                            <note>A collection of 90 <ref type="authFile" corresp="#Qene">qǝne</ref> of different kinds: 
+                                65 <ref type="authFile" corresp="#Sellase"><foreign xml:lang="gez-tr">śǝllāse</foreign></ref>, 
+                                12 <ref type="authFile" corresp="#Mawaddes"><foreign xml:lang="gez-tr">mawaddǝs</foreign></ref>, 
+                                8 <ref type="authFile" corresp="#Wazema"><foreign xml:lang="gez-tr">wāzemā</foreign></ref>,
+                                4 <ref type="authFile" corresp="#EtanaMogar"><foreign xml:lang="gez-tr">ʿǝṭāna mogar</foreign></ref> 
+                                and 1 <ref type="authFile" corresp="#Zayeeze"><foreign xml:lang="gez-tr">zayǝʾǝzeni</foreign></ref>.</note>
+                            <incipit xml:lang="gez">ናሁ፡ እጽሕፍ፡ መዝገበ፡ ድርሳን፡ ዘይረስዎ፡ አዕሩግ፡ የዋሃን፡ ወወራዙት፡ ማእምራን፡ ውስተ፡ ቅድስት፡ ቤተ፡ ክርስቲያን፡
+                                ባሕቱ፡ ይክሥ<pb n="40ra"/>ት፡ ሊተ፡ እግዚአብሔር፡ ጥበበ፡ ወኪነ፡ አሜን።</incipit>
+                            <textLang mainLang="gez"/>
+                            <msItem xml:id="ms_i2.1">
+                                <title ref="LIT7038Qene">Qǝne poem of the <foreign xml:lang="gez">śǝllāse</foreign>-type on ʾEwosṭātewos</title>
+                                <note>Probably on <persName ref="PRS3934Ewostate"/>.</note>
+                                <incipit xml:lang="gez"><persName ref="PRS3934Ewostate">ኤዎስጣቴዎስ፡</persName> እንዘ፡ እልፍ፡ ውሉድ፡ ውስተ፡ ሕፅንከ፡ 
```

Comment:
please include cert="...". Actually, we cannot be sure that this is St Ewostatewos from norther Ethiopia.